docker-compose.yml

```diff
@@ -3,15 +3,14 @@ version: "3"
 services:
   telemetry-frontend:
     build:
-      dockerfile: frontend.Dockerfile
-      context: ./
+      dockerfile: Dockerfile
+      context: ./frontend/
     # Copy in changes to the ui, so no need to rebuild the images.
     volumes:
       - /app/node_modules
       - ./packages:/app/packages
-    command: yarn start:frontend
     ports:
-      - 3000:3000
+      - 3000:80
   telemetry-backend:
     build:
       dockerfile: backend.Dockerfile
```

Comment:
AFAIU [this](https://github.com/paritytech/substrate-telemetry/blob/master/backend/Dockerfile) is the newer and better image for the backend and the `backend.Dockerfile` shall be removed. 
/cc @chevdor

Reply:
👍  that's correct.

Reply:
As far as:

>  since we are getting to publish these images in our registry, I presume, it makes sense to update LABEL in Dockerfiles and the main README.md

I am fine with that as long as credits are fair kept. Guys like @woss and myself spent quite some time working on those and that is independant from the new location for hosting the new images.

If you/parity want to take over the maintenance, the best would be syncing with @woss and make the maintainer label a list including him. You could make @woss and myself part of the authors and take over the maintenance label. 

That sounds fair to keep the original authors than to wipe the names of people who initially put things together for the sake of 'cleaning up'.

Reply:
Of course, I never meant to remove not you, nor @woss from credits. The list is a good idea!
And as for the registry, I think it's more sustainable to set up a CI and push it to a registry on every update automatically rather than manually.

Reply:
_(for clarity, this will only push to docker hub on tagged commits.)_